exporters/auto/README.md

```diff
@@ -0,0 +1,11 @@
+# Auto-Instrumentation for OpenTelemetry in Google Cloud
+
+## Setup
+
+To instrument traces using the `opentelemetry-javaagent`, `opentelemetry-operations-java-auto-<version>.jar` can be used to provide opentelemetry exporters.
+
+```
+java -javaagent:path/to/opentelemetry-javaagent-<version>-all.jar \
+     -Dotel.exporter.jar=path/to/opentelemetry-operations-java-auto-<version>.jar \
```

Comment:
I'm not familiar with the auto-instrumentation implementation, so I have a question.
This jar currently only implements the trace exporter. If we decided to also implement metrics exporter (is that even supported?), would it be part of the same Gradle project / jar or a separate one?

Having them separately would be more flexible (e.g. a customer may want to only use tracing part of Google Cloud and use a third party vendor for metrics, and vice versa), but requires more boilerplate in the repository.

Since the goal is to provide a simple "auto" mechanism, having one jar for all exporters is probably fine.

Reply:
Update: Yes, metric exporters are supported as well: [MetricExporterFactory](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/javaagent-spi/src/main/java/io/opentelemetry/javaagent/spi/exporter/MetricExporterFactory.java)

@AlexanderMann could you implement it as well? Should be very similar to trace exporter.